FeistyUITests/GameViewTests/GameViewTestCase.swift

```diff
@@ -18,7 +18,7 @@ class GameViewTestCase: XCTestCase {
     
```

Comment:
This line actually added 2 arguments ("-UseStubs" and "true") and when it should have been "-UseStubs" with the associated value of true